app/views/admin/products_v3/_table.html.haml

```diff
@@ -59,11 +59,12 @@
             %td.align-left
               .line-clamp-1= product.inherits_properties ? 'YES' : 'NO' #TODO: consider using https://github.com/RST-J/human_attribute_values, else use I18n.t (also below)
           - product.variants.each do |variant|
+            - prefix = "[products][][variants_attributes][]" # Couldn't convince the formbuilder to generate this for me, so for now we manually add the prefix
```

Comment:
Did you try `products][][variants_attributes` as name? Just a hacky idea.

Reply:
😮  That is a clever idea, I didn't think of that!

Reply:
This works! It means we no longer need to explicitly set the name and can simplify the code a lot.

```haml
            = form.fields_for("products][][variants_attributes][", variant) do |variant_form|
               ...
                  = variant_form.text_field :display_name, 'aria-label': t('admin.products_page.columns.name'), placeholder: product.name
```
Produces:
```html
<input aria-label="Name" placeholder="" type="text" value="" name="[products][][variants_attributes][][display_name]" id="_products__variants_attributes__display_name">
```

However, Rails won't let me use an explicit index with this method (https://github.com/openfoodfoundation/openfoodnetwork/pull/11468#discussion_r1312475248). Oh well, I'll take the cleaner code.